zebra-network/src/meta_addr.rs

```diff
@@ -240,6 +240,19 @@ impl MetaAddr {
         self.last_seen
     }
 
+    /// Update reported last time for `NeverAttempted` peer by applying an offset.
+    ///
+    /// This adds the provided offset to the `last_seen` time reported by the peer that gossiped
+    /// this address. This is done to compensate for any clock differences between the local node
+    /// and the node that reported the address.
+    ///
+    /// If the peer has already been interacted with by us, this method does nothing.
+    pub fn offset_last_seen_by(&mut self, offset: Duration) {
+        if self.last_connection_state == NeverAttemptedGossiped {
```

Comment:
```suggestion
        // TODO: replace with `!self.has_been_attempted()` once #2160 merges
        if self.last_connection_state == NeverAttemptedGossiped {
```

Reply:
I'd like to add this comment in this PR, because it's likely that this PR will merge before #2160.
(#2160 still has some bugs I need to diagnose and fix.)